### ISSUE TITLE
TypeError: Cannot read property 'edit' of undefined

### ISSUE THREAD

**Issue by vagnerkuntz** [ethkat/tibia-ts3-teamspeakbot#34]
TypeError: Cannot read property 'edit' of undefined

**Comment by jotavf** [comment]
Instalou yarn ou npm de boa?

**Comment by vagnerkuntz** [comment]
Yes, I installed the normal yarn ...
doing new tests I found that if you delete the channels when running ts3-setup it recreates the channels more then this error appears

**Comment by jotavf** [comment]
Make sure you've took a note of @Ethaan considerations:

> 1.- Dont delete the channel with the name of the bot (see src/utils/constants.js), because there is a chance this will break the relationship between DB and TS, if you do that, you will need to start from scratch again.

> 2.- There is a know bug on the parser for the command addEnemysByGuild and addFriendsByGuild, where some character are not added, this just happend in weird scenarios where there is an spceial rank on the guild, and is really hard or complex to get them.

**Comment by vagnerkuntz** [comment]
yes i see the note, i'm thinking about how we can fix this problem, because it recreates the tables but just can't identify her new id

**Comment by Ethaan** [comment]
@vagnerkuntz did you get into this? i'm planning on re-visit this bot, to make it more user friendly, even doing an small dashboard, so you guys don't really need to touch the teamspeak server and do everything on the web

**Comment by vagnerkuntz** [comment]
I found the problem, but I'm still not able to solve it. yes i can help to test and with basic functions

**Comment by Eduardjo** [comment]
I have the same problem. I can't start, I'm trying to configure "yarn ts3-setup". And it tells me this error:
/tibia-ts3-teamspeakbot (master)
$ yarn ts3-setup
yarn run v1.22.19
$ babel-node -r ./node_modules/dotenv/config.js ./src/scripts/setup
"babel-node" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
